### ISSUE TITLE
Bug while installing GRRLIB on Windows

### ISSUE THREAD

**Issue by Holiaaa** [GRRLIB/GRRLIB#45]
Hi, i try to install GRRLIB on Windows after compiling it and i have this issue : 

```sh
cp: target '/opt/devkitpro/portlibs/wii/include' is not a directory
make[1]: *** [Makefile:98: install] Error 1
make[1]: Leaving directory '/home/teoja/Documents/Grrlib/GRRLIB/GRRLIB/GRRLIB'
make: *** [Makefile:15: install] Error 1
```

Can you help me please ?

**Comment by Crayon2000** [comment]
Make sure the user running the `make install` command has write access to the _$DEVKITPRO/portlibs_ folder or the installation will fail.

**Comment by Crayon2000** [comment]
I have also done a fix in 8ca7a33402a198a5d9405f8bb19a57e59a13867d to always create the folders if they are not always there. Please use the master branch and try again. Let me know if it worked.
